packages/gestalt/src/TextArea.js

```diff
@@ -91,28 +99,69 @@ const TextAreaWithForwardRef: React$AbstractComponent<
     styles.textArea,
     formElement.base,
     disabled ? formElement.disabled : formElement.enabled,
-    hasError || errorMessage ? formElement.errored : formElement.normal
+    (hasError || errorMessage) && !focused
+      ? formElement.errored
+      : formElement.normal,
+    tags
+      ? {
+          [focusStyles.accessibilityOutlineFocus]: focused,
+          [styles.textAreaWrapper]: true,
+        }
+      : {}
+  );
+
+  const inputElement = (
+    <textarea
+      aria-describedby={errorMessage && focused ? `${id}-error` : null}
+      aria-invalid={errorMessage || hasError ? 'true' : 'false'}
+      className={tags ? styles.unstyledTextArea : classes}
+      disabled={disabled}
+      id={id}
+      name={name}
+      onBlur={handleBlur}
+      onChange={handleChange}
+      onFocus={handleFocus}
+      onKeyDown={handleKeyDown}
+      placeholder={placeholder}
+      ref={ref}
+      rows={tags ? undefined : rows}
+      value={value}
+    />
   );
 
+  const tagsWrapperStyle = {
+    height: rows * ROW_HEIGHT + INPUT_PADDING_WITH_TAGS,
+  };
```

Comment:
I had to manually size the component based on number of rows, since the sizing is on the div. Not sure if this would be the preferred way of doing this

Reply:
Can we check with design what's the right height for this?

Default is 3 rows, but only two rows of tags are visible.

 ![image](https://user-images.githubusercontent.com/10593890/102555926-fb3ef500-4084-11eb-816e-16fb85bc914e.png)

Reply:
Also do we need placeholder if we have Tags? Maybe the example could remove the placeholder after Tags are created.

![image](https://user-images.githubusercontent.com/10593890/102556981-6b4e7a80-4087-11eb-8886-e7e251201337.png)

Reply:
@ashleyseto what do you think about the height and placeholder options here?

Reply:
> Default is 3 rows, but only two rows of tags are visible.

it's 3 rows of text, which ends up being 2 rows of text 😕  Another option could be to have the rows as a min height, and just grow from there instead of scrolling

Reply:
I think if someone wants to used text area, then ideally they're setting a height and then it expands from there so yeah a min-height makes sense.

I know also I added a placeholder to continue, I think I could update the designs so the placeholder doesn't keep appearing after 1 tag is added.

Reply:
@ashleyseto @ohariz Just to wrap up this change: `rows` still sets a (min)height that doesn't match with the number of Tag rows, however, TextArea grows as new lines are added.  

![image](https://user-images.githubusercontent.com/10593890/103942021-bb0be900-50fd-11eb-8f4b-da7d48503330.png)

Reply:
Also, quick one. Could you specify in the Docs that the rows props `Number of text (not Tag) rows to display.`   
or something like that?